### ISSUE TITLE
Request: add whisper functionality

### ISSUE THREAD

**Issue by 741N** [vinszent/gnome-twitch#324]
as it says on the tin: is there a chance of whispers being added, aka private messages? I just logged on the site after a while and noticed I'd gotten a whisper.

**Comment by vinszent** [comment]
Yeah sure I can look into it. However it doesn't run off IRC so it might take me a while to work it into the rest of the app.
